interp/notation.ml

```diff
@@ -1377,28 +1378,28 @@ let rec insert_coercion_path path = function
       else if shorter_path path path' then path::allpaths
       else path'::insert_coercion_path path paths
 
-let declare_entry_coercion (scope,(entry,_) as specific_ntn) entry' =
-  let entry, lev = decompose_custom_entry entry in
+let declare_entry_coercion (scope,(entry,key)) entry' =
+  let lev = None in
```

Comment:
The level should not be discarded. It should instead take the `entry_level` found by looking at `level_of_notation`. Probably not here, but at the call place of `declare_entry_coercion`.

Reply:
Yes, the problem for this and the other occurrence is that the level map is not available from `interp` (only from `parsing`). I could move it, but unfortunately it also contains the parsing rules (which belong to `parsing`, where they currently are). I could also split in two maps.

Reply:
One possibility could be to split the table into one for the parsing rule and another for the level. It would not make a big difference.